package.json

```diff
@@ -131,7 +132,8 @@
     ],
     "setupFiles": [
       "./shim.js"
-    ]
+    ],
+    "snapshotSerializers": ["enzyme-to-json/serializer"]
```

Comment:
what does that bring?

Reply:
It should make sure that the snapshots made with enzyme methods like shallow are also looking JSX, like the raw jsx would be I think

see [yarn.pm/jest-serializer-enzyme](https://yarn.pm/jest-serializer-enzyme)

Reply:
@Haroenv says right (the original package is [this one](https://github.com/adriantoine/enzyme-to-json))

It allows you to do:

```js
import { shallow } from 'enzyme';

const wrapper = shallow(MyComponent);

expect(wrapper).toMatchSnapshot();
```

We don't rely on `react-test-renderer` inside the test anymore. You can handle all your tests on components with enzyme (Snapshot + assertions).